songs.py

```diff
@@ -29,6 +29,61 @@ def get_spotify_genres():
     return genres
 
 
+def get_similar(song, limit):
+    SPOTIFY_API_KEY = get_spotify_token()
+    headers = {"Authorization": f"Bearer {SPOTIFY_API_KEY}"}
+
+# FIRST API CALL, GRAB ID OF USER'S SONG:
+
+    params = {
+        "q": song,
+        "type": "track",
+        "limit": 1
+    }
+
+    response = requests.get("https://api.spotify.com/v1/search", headers=headers, params=params)
```

Comment:
The base of this api url is **https://api.spotify.com/v1/** . You're using it here and down below in the 2nd api call. You can perhaps save this as a constant string variable and then just add the endpoints like /search